pkg/backup/backup.go

```diff
@@ -71,6 +72,9 @@ const BackupVersion = 1
 // BackupFormatVersion is the current backup version for Velero, including major, minor, and patch.
 const BackupFormatVersion = "1.1.0"
 
+// ArgoCD managed by namespace label key
+const ArgoCDManagedByNamespaceLabel = "argocd.argoproj.io/managed-by"
```

Comment:
Found the key to be from argocd operator.
https://github.com/argoproj-labs/argocd-operator/blob/33344872c0db34dc0250075bb4d707fee2246c60/common/keys.go#L95